File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioVarMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdVar;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioVarMap extends StudioBase
{
  public MetaIdVar[] keys;

  public Map<MetaIdVar, StudioVar> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioVarMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioVar

interface StudioVarMap : StudioBase
{
  val keys: Array<MetaIdVar>
  val map: Map<MetaIdVar, StudioVar>
}